### PR TITLE
add new routes for anaplan features

### PR DESCRIPTION
**Pull request by badr96**

### CODE REVIEW

File: HyperAPI/hdp_api/routes/anaplan.py [hunk 1]
```diff
@@ -25,7 +25,20 @@ class _getFiles(Route):
             'workspace_id': Route.VALIDATOR_OBJECTID,
             'model_id': Route.VALIDATOR_OBJECTID
         }
-        
+    class _getImports(Route):
```
Comment (MichaelVasseurBE) [inline HyperAPI/hdp_api/routes/anaplan.py]: Missing decorator `available_since`

File: HyperAPI/hdp_api/routes/anaplan.py [hunk 2]
```diff
@@ -25,7 +25,20 @@ class _getFiles(Route):
...
+        name = "getImports"
+        httpMethod = Route.GET
+        path = "/anaplan/workspaces/{workspace_id}/model/{model_id}/imports/{dataset_Id}"
+        _path_keys = {
+            'workspace_id': Route.VALIDATOR_OBJECTID,
+            'model_id': Route.VALIDATOR_OBJECTID,
+            'dataset_Id': Route.VALIDATOR_OBJECTID
+        }
+    class _sendPayload(Route):
```
Comment (MichaelVasseurBE) [inline HyperAPI/hdp_api/routes/anaplan.py]: Missing decorator `available_since`

File: HyperAPI/hdp_api/routes/anaplan.py [hunk 3]
```diff
@@ -25,7 +25,22 @@ class _getFiles(Route):
...
+        httpMethod = Route.GET
+        available_since = '7.0.2'
+        path = "/anaplan/workspaces/{workspace_id}/model/{model_id}/imports/{dataset_Id}"
+        _path_keys = {
+            'workspace_id': Route.VALIDATOR_OBJECTID,
+            'model_id': Route.VALIDATOR_OBJECTID,
+            'dataset_Id': Route.VALIDATOR_OBJECTID
+        }
+    class _sendPayload(Route):
+        name = "SendPayload",
```
Comment (serkansahin) [inline HyperAPI/hdp_api/routes/anaplan.py]: "sendPayload" be careful ;)

File: HyperAPI/hdp_api/routes/anaplan.py [hunk 4]
```diff
@@ -25,7 +25,22 @@ class _getFiles(Route):
...
+        available_since = '7.0.2'
```
Comment (serkansahin) [inline HyperAPI/hdp_api/routes/anaplan.py]: availabe_since = '4.2.6'

**Pull request merged**